jetty-util/src/main/java/org/eclipse/jetty/util/log/AbstractLogger.java

```diff
@@ -20,6 +20,8 @@
 
 import java.util.Properties;
 
+import org.eclipse.jetty.util.StringUtil;
```

Comment:
Not sure I like this... I would prefer the logging to depend only on JDK classes.
So I would just inline `StingUtil.isEmpty()` where needed.